instances/treasury-factory.near/widget/components/ProfileAutocomplete.jsx

```diff
@@ -0,0 +1,136 @@
+const { onClose, onSelect } = props;
+if (!context.accountId || !props.term) return <></>;
+
```

Comment:
we do have a autocomplete component in [devhub](https://github.com/NEAR-DevHub/neardevhub-bos/blob/main/instances/devhub.near/widget/devhub/entity/proposal/AccountInput.jsx)